doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -788,6 +788,7 @@ Bluetooth libraries and services
     * A new function :c:func:`bt_fast_pair_fmdn_is_provisioned` for the FMDN extension API.
       This function can be used to synchronously check the current FMDN provisioning state.
       For more details, see the :ref:`ug_bt_fast_pair_gatt_service_fmdn_info_callbacks_provisioning_state` section in the Fast Pair integration guide.
+    * The :ref:`cs_de_readme` library.
```

Comment:
This must be the first entry under the Bluetooth libraries and services heading like this:

* Added the :ref:`cs_de_readme` library.